<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLugaresConfigPage</class>
 <widget class="QRecordWidgetMapper" name="FLugaresConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecordWidgetMapper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Lugares:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TListViewModelRecord" name="lvwLugares">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnNuevo">
         <property name="toolTip">
          <string>Nuevo lugar</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/document-new.png</normaloff>:/pics/document-new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAtras">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Lugares anteriores</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/atras.png</normaloff>:/pics/atras.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdelante">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Siguientes lugares</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/adelante.png</normaloff>:/pics/adelante.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Lugar</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ledNombre"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dirección</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ledDireccion"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Télefono</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ledTelefono"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Tipo</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="TComboBoxModelRecord" name="cbmrTipo"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Empresa</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="TComboBoxModelRecord" name="cbmrEmpresa"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>177</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnGuardar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="icon">
      <iconset resource="../../app/app.qrc">
       <normaloff>:/pics/save.png</normaloff>:/pics/save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnCancelar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="../../app/app.qrc">
       <normaloff>:/pics/dialog-close.png</normaloff>:/pics/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>146</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TComboBoxModelRecord</class>
   <extends>QComboBox</extends>
   <header>tcomboboxmodelrecord.h</header>
  </customwidget>
  <customwidget>
   <class>TListViewModelRecord</class>
   <extends>QListView</extends>
   <header>tlistviewmodelrecord.h</header>
  </customwidget>
  <customwidget>
   <class>QRecordWidgetMapper</class>
   <extends>QWidget</extends>
   <header>qrecordwidgetmapper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../app/app.qrc"/>
  <include location="../../app/app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FLugaresConfigPage</sender>
   <signal>hasRecordsWithOutSave(bool)</signal>
   <receiver>btnGuardar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FLugaresConfigPage</sender>
   <signal>hasRecordsWithOutSave(bool)</signal>
   <receiver>btnCancelar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lvwLugares</sender>
   <signal>navPrevious(bool)</signal>
   <receiver>btnAtras</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lvwLugares</sender>
   <signal>navNext(bool)</signal>
   <receiver>btnAdelante</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAtras</sender>
   <signal>clicked()</signal>
   <receiver>lvwLugares</receiver>
   <slot>navToPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdelante</sender>
   <signal>clicked()</signal>
   <receiver>lvwLugares</receiver>
   <slot>navToNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
